pkg/minikube/config/extra_options.go

```diff
@@ -38,6 +40,29 @@ type ExtraOptionSlice []ExtraOption
 // ComponentExtraOptionMap maps components to their extra opts, which is a map of keys to values
 type ComponentExtraOptionMap map[string]map[string]string
 
+// Exists returns true if component.key (parsed from value) is already in ExtraOptionSlice
+func (es *ExtraOptionSlice) Exists(value string) bool {
```

Comment:
This looks good! Would you mind writing a unit test for this in `extra_options_test.go`?

Reply:
Added unit tests...